### PR TITLE
starting to bench structural models

### PR DESCRIPTION
**Pull request by guilhermebodin**
We must start keeping track of structural model benchmarks

### CODE REVIEW

File: benchmark/bench.jl [hunk 1]
```diff
@@ -46,4 +46,40 @@ Q = fill(1.0, (1,1))
...
+# BenchmarkTools.Trial: 
+#   memory estimate:  5.45 GiB
+#   allocs estimate:  11321246
+#   --------------
+#   minimum time:     4.361 s (8.57% GC)
+#   median time:      4.464 s (8.60% GC)
+#   mean time:        4.464 s (8.60% GC)
+#   maximum time:     4.567 s (8.62% GC)
+#   --------------
+#   samples:          2
```
Comment (raphaelsaavedra) [inline benchmark/bench.jl]: I'm not super familiar with BenchmarkTools, is there a way to require more samples?

Comment (guilhermebodin) [reply benchmark/bench.jl]: hmm i don`t think so

Comment (guilhermebodin) [reply benchmark/bench.jl]: this is huge because it never attains steadystate

Comment (guilhermebodin) [reply benchmark/bench.jl]: We should change the `check_steady_state` function to see if this equation holds
Captura de Tela 2019-08-12 às 20 09 56

Comment (guilhermebodin) [reply benchmark/bench.jl]: I am almost sure that this is 
`P = TPttT' + RQR'`

Comment (raphaelsaavedra) [reply benchmark/bench.jl]: That's a nice find! Also, even if it legitimately does not attain steady state, I think we're still using much more memory than we should. Need to investigate what's causing so much trouble, perhaps use a TimerOutputs branch

Comment (guilhermebodin) [reply benchmark/bench.jl]: sounds like a good idea!

**Pull request merged**